modules/core/persistence_postgres.xml

```diff
@@ -24,6 +24,7 @@
 		<context:exclude-filter type="regex"
 			expression="org.opensrp.connector..*" />
 	</context:component-scan>
+	<context:component-scan base-package="org.smartregister.stock.openlmis" />
```

Comment:
This should be optional, define this inside an openlmis profile

Reply:
removed this